lib/db/json-converter.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports.fromText = it => {
+    if (typeof it === 'string') {
+        return JSON.parse(it);
+    }
+    return it;
+};
```

Comment:
this is so open. it really needs guards to centralize the logic. If I send through "null", or "{" I think even "}" it'll blow up I think

Reply:
Yes, you're absolutely right. Implicitly, the schema is being put into the DB is valid JSON (mysql and postgres validate this), but we should definitely be explicit about deserialization issues.